.github/workflows/build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: Gradle Build and Test
```

Comment:
```suggestion
      - name: Gradle Build
```

Reply:
The `build` task runs also the tests. For this reason I removed the `test` one.